django/db/__init__.py

```diff
@@ -59,3 +62,13 @@ def close_old_connections(**kwargs):
 
 signals.request_started.connect(close_old_connections)
 signals.request_finished.connect(close_old_connections)
+
+
+# Register an event to close all connections before forking a process.
+# Only register if the platform supports forking.
+def close_all_connections(**kwargs):
+    connections.close_all()
+
+
+if platform.system() == "Linux":
+    os.register_at_fork(before=close_all_connections)
```

Comment:
By looking at the API in more details I think we'll want to use `after_in_child` here instead as we want to keep the connection in the parent opened.

To make it clear, the current code closes the connection in the parent process before takes place so the connection is effectively closed in the child when `connection.is_usable()` is called.